Formula/tssh.rb

```diff
@@ -0,0 +1,18 @@
+class Tssh < Formula
+  desc "SSH Lightweight management tools"
+  homepage "https://github.com/luanruisong/tssh"
+  url "https://github.com/luanruisong/tssh/archive/refs/tags/1.2.0.tar.gz"
+  sha256 "84c2db9e0ee00350e3c998d5666ed587a98bd4f2650a1166c7532cfffbb48815"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"tssh", "-ldflags", "-X main.version=#{version}"
+  end
+
+  test do
+    status_output = shell_output("#{bin}/tssh -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula